pkg/metrics/metrics.go

```diff
@@ -27,30 +27,37 @@ var (
 			Help: "A count of total failed requests received by the OCM Agent service",
 		}, []string{})
 
-	metricRequestFailure = prometheus.NewGaugeVec(
+	MetricRequestFailure = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "ocm_agent_request_failure",
 			Help: "Indicates that OCM Agent could not successfully process a request",
 		}, []string{"path"})
 
-	metricResponseFailure = prometheus.NewGaugeVec(
+	MetricResponseFailure = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "ocm_agent_response_failure",
 			Help: "Indicates that the call to the OCM service endpoint failed",
-		}, []string{"service"})
+		}, []string{"ocm_service"})
+
+	metricServiceLogSent = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "ocm_agent_service_log_sent",
+			Help: "A count of total service log sent based on managedNotification template",
+		}, []string{"ocm_service", "template", "state"})
 
 	metricsList = []prometheus.Collector{
```

Comment:
some var names are capitalized while others not, is this intentional?

Reply:
yes, the ones capitalized need to be called in the other packages. Mainly used for reset the metric once the failure condition finished.

Reply:
example: https://github.com/openshift/ocm-agent/pull/29/files#diff-76c4aa4812efc417827a4df668f5be466249c5511c82029c475de955ea854a1dR194